packages/ckeditor5-build-helpjuice-v2/src/plugins/link/src/linkcommand.ts

```diff
@@ -150,7 +150,7 @@ export default class LinkCommand extends Command {
 	 * @param href Link destination.
 	 * @param manualDecoratorIds The information about manual decorator attributes to be applied or removed upon execution.
 	 */
-	public override execute( href: string, text: string, manualDecoratorIds: Record<string, boolean> = {} ): void {
+	public override execute( href: string, text: string | null = null, manualDecoratorIds: Record<string, boolean> = {} ): void {
```

Comment:
Why do we need this change?

Reply:
After rebasing, I get two TS errors when running `npm run build`

> Adding an optional value to the text parameter fixed the issue for me

![Screenshot from 2023-07-20 16-30-40](https://github.com/Helpjuice/ckeditor5/assets/45231276/b4384848-e1ba-4012-ab35-c6ae69e314ba)

Reply:
@SpaYco

Any thoughts on this? Looks like it's related to the links PR

Reply:
@NicholasJarr looks like CKFinder and CKBox are throwing those errors after my changes to links plugin, the problem is that i didn't get those errors on my end when i build the app, but @Bondok6 receives them. i'm not sure we use those plugins, but it's okay to have the text parameter optional as I have fail safe to use `href` if `text` is null.